posthog/api/plugin.py

```diff
@@ -354,15 +352,30 @@ def job(self, request: request.Request, **kwargs):
         job_payload = job.get("payload", {})
         job_op = job.get("operation", "start")
 
-        celery_app.send_task(
-            name="posthog.tasks.plugins.plugin_job",
-            queue=settings.PLUGINS_CELERY_QUEUE,
-            args=[self.team.pk, plugin_config_id, job_type, job_op, job_payload],
+        payload_json = json.dumps(
+            {
+                "type": job_type,
+                "payload": job_payload | {"$operation": job_op},
+                "pluginConfigId": plugin_config_id,
+                "pluginConfigTeam": self.team.pk,
+            }
         )
+        sql = f"SELECT graphile_worker.add_job('pluginJob', '{payload_json}')"
+        execute_postgres(sql)
 
         return Response(status=200)
 
 
+def execute_postgres(sql):
+    from django.db import connection
+
+    try:
+        with connection.cursor() as cursor:
+            cursor.execute(sql)
+    except Exception as e:
+        raise Exception(f"Failed to execute postgres sql={sql},\nexception={str(e)}")
```

Comment:
This function is super generic but at the same doesn't really offer any more utility than plain `with connection.cursor() as cursor:`. I suppose it's just to make spying easier. But could we maybe avoid this by spying on `posthog.api.plugin.connection`?